CoreFoundation/Base.subproj/CoreFoundation.h

```diff
@@ -33,6 +35,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <netdb.h>
```

Comment:
So there are two CoreFoundation.h's one for swift and one for Darwin. this one is the Darwin version.

Having alterations to this header could prove to be a non starter because it would prevent us from merging changes back from CoreFoundation. That being said if the reason is compelling enough for us to change the header for ALL platforms then perhaps it can be done, but I think it would be best to tackle it separately if possible.